modules/common/tests/src/Unit/DataResourceTest.php

```diff
@@ -18,21 +18,6 @@
  */
 class DataResourceTest extends TestCase {
 
-  /**
```

Comment:
We should leave this test because even though it covers deprecated code, the code is still there.

Reply:
I was having trouble getting the test to work and thought it was maybe not worth it but I'll give it another shot

Reply:
Actually it just works as is, I'm not sure why this was removed tbh! Restored.